consensus/clique/clique.go

```diff
@@ -26,6 +26,9 @@ import (
 	"sync"
 	"time"
 
+	lru "github.com/hashicorp/golang-lru"
+	"golang.org/x/crypto/sha3"
```

Comment:
The import order is whatever `goimports` gives us. I prefer that vs manual groups.